data/json/recipes/armor/suit.json

```diff
@@ -857,8 +843,8 @@
     "time": "6 h",
     "autolearn": true,
     "book_learn": [ [ "textbook_armwest", 3 ] ],
-    "using": [ [ "sewing_standard", 35 ], [ "fastener_large", 1 ] ],
-    "components": [ [ [ "rag", 26 ] ] ]
+    "using": [ [ "sewing_standard", 30 ], [ "fabric_standard_nostretch", 26 ] ]
+    "components": [ [ [ "rag", 26 ] ], [ "fastener_large", 1 ] ]
```

Comment:
```suggestion
    "using": [ [ "sewing_standard", 35 ], [ "fabric_standard_nostretch", 26 ], [ "fastener_large", 1 ] ]
```

Reply:
@Maleclypse I think this change was lost when you applied your changes... The fastener should be in "using" and the rags are already included in fabric_standard_nostretch.